.travis-old.yml

```diff
@@ -8,7 +8,7 @@ cache:
     - $HOME/.phantomjs
```

Comment:
Feel free to just remove this file altogether, we can always find it in the history :>

Reply:
I was mostly keeping this as a reference for myself while working on this PR. Now that we have feature parity, I can indeed remove it :D